IPython/lib/latextools.py

```diff
@@ -146,11 +146,11 @@ def latex_to_png_dvipng(s, wrap, color='Black', scale=1.0):
         return None
     try:
         workdir = Path(tempfile.mkdtemp())
-        tmpfile = workdir.joinpath("tmp.tex")
-        dvifile = workdir.joinpath("tmp.dvi")
-        outfile = workdir.joinpath("tmp.png")
+        tmpfile = "tmp.tex"
+        dvifile = "tmp.dvi"
```

Comment:
Does this mean that this file is no longer "temporary"? I.e., it doesn't get removed by the `shutil.rmdir` in line 189?

Reply:
I think it's the case of missing `workdir.joinpath(...)` in line 175.

Reply:
All files generated are still temporary. The `dvifile` from line 175 is generated by `dvips` in the `workdir` (it not, line 184 would fail) and is removed at line 189.